src/hts/home/summary-tiles/hts-summary-tiles.component.tsx

```diff
@@ -24,7 +26,7 @@ function HTSSummaryTiles({ launchWorkSpace }) {
       subTitle: 'Active Visits',
```

Comment:
could make this more generic to just take in values.

Reply:
That is a string from the design mockups and isn't dynamic. That field is also part of an array of objects passed to the Tile Component.